spec/acceptance/basic_spec.rb

```diff
@@ -37,6 +37,10 @@ class { '::foreman_proxy':
 
   it_behaves_like 'a idempotent resource'
 
+  describe package('foreman-proxy-journald') do
+    it { is_expected.not_to be_installed }
+  end
+
```

Comment:
Saw this in `foreman` codebase, not sure if it is the correct place here.

Reply:
It's the right place.